src/Mutator/Operator/Concat.php

```diff
@@ -93,7 +93,7 @@ public function mutate(Node $node): iterable
 
         $newNode = new Node\Expr\BinaryOp\Concat($left, $right);
 
-        /** @psalm-suppress ImpureMethodCall */
```

Comment:
it was a correct way. You can check it locally with `make psalm`

Reply:
yeah, I did, it was right, but CS failed :(

Reply:
see: https://github.com/infection/infection/runs/4059480272?check_suite_focus=true